### DISCUSSION TITLE
Add sign support and book support in chatfilter.yml

### DISCUSSION THREAD

**Discussion by goosBanny** [Zrips/CMI#7043] [Enhancement]
Make the chat filter check all text in books and on signs.

**Comment by mrfloris** [comment]
I wish the chat filter would indeed extend to nameplate for mobs, signs, and books, as well as names for items through /anvil and nametag. Maybe even usernames. 
so someone can't make a pack of wolves with hate speech names and sit them around spawn. or make drug related items in the anvil, or write certain profanity in books and signs. And that someone with a blacklisted name will get shadowmuted or kickedbanned when they join with it.

go beyond chat filter, sure has my vote ! 

Additioanlly, if we could add #comments around the chat filter regex, so we can make it more human readable, that would be great. that way we can say
#hate speech words:
regex
regex
# sexual word
regex
regex

and still understand what it is filtering.
